### ISSUE TITLE
Accessibility: Screen reader doesn't announce the Ordering for will be installed and feature list item.

### ISSUE THREAD

**Issue by a-gazin** [wixtoolset/issues#6390]
## Bugs

If this issue is a bug:

* Which version of WiX are you building with?

> (WiX version in major.minor.release.build format)
![Wix_Version](https://user-images.githubusercontent.com/61299470/110696226-aa303000-819f-11eb-9558-4eb74093cbe1.png)

* Which version of Visual Studio are you building with (if any)?

> We don't use Visual Studio. We use MSBuild version 16.4.0.56107

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

> N/A

* Which version of .NET are you building with?

> 3.5

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> Windows 10 Version 20H2 (OS Build 19042.804)

* Describe the problem and the steps to reproduce it.

> This is an accessibility issue. To reproduce, you need to have NVDA (2019.2.1) screenreader installed:
https://www.nvaccess.org/download/

Repro steps:
1. Select and run msoledbsql.msi file. The installer can be downloaded from here: https://docs.microsoft.com/en-us/sql/connect/oledb/download-oledb-driver-for-sql-server?view=sql-server-ver15#download
2. "Welcome to the OLE DB Driver for SQL server Setup" screen will open.
3. Select and press "Next" button, License agreement screen should open.
4. Select "I accept the terms in License agreement" radio button and press next button.
5. Feature Selection screen should open.
6. Start screenreader and navigate using down arrow key to the OLEDB driver dropdown and expand it then check whether its announce the ordering or not for list item.

* Describe the behavior you expected and how it differed from the actual behavior.
The following video provided by the accessibility engineer explains the issue and repro steps:
https://user-images.githubusercontent.com/61299470/110697309-0d6e9200-81a1-11eb-93cf-286885b68826.mp4

> Expected
Screenreader should notify the user about the ordering of the feature selection dropdown context menu.

>Actual
Screenreader does not notify the user about the ordering of the feature selection dropdown context menu. This is likely an accessibility issue with the [Control element of type SelectionTree](https://wixtoolset.org/documentation/manual/v3/xsd/wix/control.html) in WiX. It seems that internally context menus are used instead of drop downs for such elements.

**Comment by barnson** [comment]
This package isn't using the WixUI dialog library. If Windows Installer doesn't support the accessibility behavior you're looking for, there's nothing WiX can do to add it.
